### ISSUE TITLE
Flowfuse Dashboard 2 doesn't hide chrome searchbar on mobile

### ISSUE THREAD

**Issue by joepavitt** [FlowFuse/node-red-dashboard#972]
### Description

https://www.reddit.com/r/nodered/comments/1df7sh5/flowfuse_dashboard_2_doesnt_hide_chrome_searchbar/

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

**Comment by colinl** [comment]
You have to use Add to Home Screen or Install as App or similar to create the shortcut.

**Comment by joepavitt** [comment]
Yep - thanks @colinl - bit confused as to why the original Dashboard would have had the bar hidden though.

**Comment by colinl** [comment]
It wouldn't, unless he/she did Add to Home Screen.
